java/connector-node/risingwave-sink-jdbc/src/main/java/com/risingwave/connector/jdbc/PostgresDialect.java

```diff
@@ -112,6 +119,12 @@ public void bindInsertIntoStatement(
                     stmt.setArray(
                             i + 1, conn.createArrayOf(fieldType.getTypeName().name(), objArray));
                     break;
+                case VARCHAR:
```

Comment:
We may pass an array of type id to get the id via index `i` instead of checking the map via column name for each row.